swarm/network/stream/delivery_test.go

```diff
@@ -530,7 +529,8 @@ func testDeliveryFromNodes(t *testing.T, nodes, conns, chunkCount int, skipCheck
 		//now we can actually upload a (random) file to the round-robin store
 		size := chunkCount * chunkSize
 		log.Debug("Storing data to file store")
-		fileHash, wait, err := roundRobinFileStore.Store(ctx, io.LimitReader(crand.Reader, int64(size)), int64(size), false)
+		b := testutil.RandomBytes(1, size)
+		fileHash, wait, err := roundRobinFileStore.Store(ctx, bytes.NewReader(b), int64(size), false)
```

Comment:
Why not use `testutil.RandomReader()` in this case? (there are more of these cases further down the diff)

Reply:
true

Reply:
Fixed everywhere except. `swarm/storage/filestore_test.go:testFileStoreRandom` - this test looks a bit weird for me. `newTestDbStore` using `ioutil.TempDir` inside, but test cleanup `/tmp` folder. Also i'm not sure why test writing files like `/tmp/slice.bzz.16M`. Looks like this test need to cleanup, I prefer to do it separately.

Reply:
you mean `tmp/bzz`? I agree, there's something not too obvious about that test. And yes, it leaves behind garbage. Please fix. I'm fine with separate PR for that.